### DISCUSSION TITLE
Edit icecast Configuration

### DISCUSSION THREAD

**Discussion by ghost** [AzuraCast/AzuraCast#5032] [Q&A]
hello everyone, please tell me, earlier it was possible to make changes to the icecast config from the web interface, today I can’t find this feature (I use the latest release build

**Comment by Vaalyn** [comment]
First you need to enable the `Advanced Features` in order for the system to show you those fields: https://docs.azuracast.com/en/administration/advanced-features

Then you can add custom Icecast config in 2 places:

For general Icecast server configs you can add those in the Station Profile in the Broadcast section and for mount point specific configs you need to add them when editing the respective mount point in the `Advanced` section of the window.

**Reply by ghost** [reply]
@Vaalyn all inclusive, it’s just not explicitly indicated that this is for editing the icecast config, but it was indicated earlier, but you won’t immediately understand what kind of xml and where to make changes, and therefore I’d better ask than break something)))
![Screenshot_2022-01-26-19-05-37-135_com miui gallery](https://user-images.githubusercontent.com/26243386/151200244-4d81505c-05f8-483b-af07-da3436f688c4.jpg)
